src/Juvix/Data/Ape/Base.hs

```diff
@@ -136,10 +136,10 @@ toCape = \case
             }
           where
             go :: a -> Bool -> Ape a -> NonEmpty (Link a)
-            go prevOp prevIsComma = \case
+            go prevOp prevIsDelimiter = \case
               ApeInfix (Infix fx' l' op' isComma' r')
-                | fx == fx' -> pure (Link prevOp prevIsComma (toCape l')) <> go op' isComma' r'
-              e -> pure (Link prevOp prevIsComma (toCape e))
+                | fx == fx' -> pure (Link prevOp prevIsDelimiter (toCape l')) <> go op' isComma' r'
+              e -> pure (Link prevOp prevIsDelimiter (toCape e))
```

Comment:
isComma' -> isDelimiter' ?

Reply:
my bad, the search and replaced was incomplete